src/wallet/wallet.cpp

```diff
@@ -1683,7 +1683,7 @@ CAmount CWalletTx::GetCredit(const isminefilter& filter) const
     return credit;
 }
 
-CAmount CWalletTx::GetImmatureCredit(bool fUseCache) const
+CAmount CWalletTx::GetImmatureCredit(const bool fUseCache) const
```

Comment:
To get signature consistency with `GetImmatureWatchOnlyCredit`. See the changes made to the same file.